File: AppSorteio/app/src/main/java/com/vieira/kassio/appsorteio/MainActivity.java
package com.vieira.kassio.appsorteio;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    Button btn;
    TextView txt;
    EditText ed;

    String participante;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn = (Button) findViewById(R.id.btnSortear);
        txt = (TextView) findViewById(R.id.txtNumber);
        ed = (EditText) findViewById(R.id.edValor);

        btn.setOnClickListener(this);
    }

    @Override
    public void onClick(View view) {

        participante = ed.getText().toString();
        if(!participante.isEmpty()){
            Sortear sortear = new Sortear(btn,txt);
            sortear.execute(Integer.parseInt(participante));
        }else{
            Toast.makeText(this,"Informe um valor",Toast.LENGTH_SHORT).show();
        }

    }
}

File: TaskKotlin/app/src/main/java/com/vieira/kassio/taskkotlin/MainActivity.kt
package com.vieira.kassio.taskkotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

     var valor : String = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        valor = edvalor.text.toString();

        btnContar.setOnClickListener {
            Tarefa(btnContar,txtnumber).execute();
        }
    }
}
